var/spack/repos/builtin/packages/libarchive/package.py

```diff
@@ -21,7 +21,6 @@ class Libarchive(AutotoolsPackage):
 
     depends_on('zlib')
     depends_on('bzip2')
-    depends_on('lzma')
```

Comment:
do you want to narrow this down for a specific `libarchive` version?

Reply:
I'm not sure that would be necessary. `xz` supersedes `lzma`, and `libarchive` still depends on it.

Reply:
now that I merged it. Actually, we should implement the `configure` line explicitly to make sure no system-`lzma` is picked up...

Reply:
From what I understand, the `liblzma.*` libraries and headers are provided by the `xz` package in modern distributions (see e.g. https://packages.debian.org/stretch/liblzma-dev). I would be surprised if the original lzma package is still installed anywhere.

Reply:
Ok, seams save then :)